plugin/pkg/scheduler/util/utils.go

```diff
@@ -39,3 +42,37 @@ func GetUsedPorts(pods ...*v1.Pod) map[int]bool {
 	}
 	return ports
 }
+
+// GetPodPriority return priority of the given pod.
+func GetPodPriority(pod *v1.Pod) int32 {
+	if pod.Spec.Priority != nil {
+		return *pod.Spec.Priority
+	} else {
+		// When priority of a running pod is nil, it means it was created at a time
+		// that there was no global default priority class and the priority class
+		// name of the pod was empty. So, we resolve to the static default priority.
+		return scheduling.DefaultPriorityWhenNoDefaultClassExists
+	}
+}
+
+// SortableList is a list that implements sort.Interface.
+type SortableList struct {
```

Comment:
You are probably right, but I don't know any better place, something like a utility library for all components.